### PR TITLE
First attempt to fix issue #227

### LINKED ISSUE #227: Meeting pages: improve top nav bar
On meeting pages like https://oscar.computeralgebra.de/meetings/2022-02/ we should a nav bar at the top for the various subpages. This could be improved in various ways:
- highlight the currently active page
- render it nicer
- allow for an order other than alphabetical (?)

### PR DESCRIPTION
**Pull request by aaruni96**
Eventually fixes #227 

Does this look better @fingolfin  ?

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ title: Home
 * [LEAN meets MaRDI and OSCAR (5 - 6 Dec, 2024 at TU Berlin).](https://polymake.org/doku.php/workshops/lean_workshop1224)
 * [Computational Algebraic Geometry Workshop (18 - 22 Nov, 2024 at Durham University).](https://sites.google.com/view/durhamcompalggeom/home)
 
+{% for e in site.data.events %}
+  
+{% endfor %}
```
Comment (fingolfin) [inline index.md]: ?

Comment (aaruni96) [reply index.md]: Whoops, this should not be here. (Started writing some code for trying to automate the upcoming events, but decided to look at the low hanging fruit of of issue 227 first)

**Comment by fingolfin** [comment]
Let's find out how it looks..

**Comment by fingolfin** [comment]
So it looks like this now for me: Screen Shot 2024-10-25 at 14 38 50

Some remarks: instead of an "aqua" box to highlight current selection, perhaps simply change it to bold face?

And the gray next to gray isn't too nice. Maybe try orange, e.g. #fab914 and then change the link color *in the top bar only* from blue to black?

**Pull request merged**